frontend/src/_includes/partials/actions.html

```diff
@@ -0,0 +1,72 @@
+<div class="mb-1 flex space-x-1">
+  <!-- Download -->
+  <a
+    href="{{ audioFileId | audio_action_download }}"
```

Comment:
Don't pollute the names of the identifiers with redundant specifiers. The context of this template is one single audio. The parameter name semantically is `fileId`.

Also, now I see that it may be proper to pass the whole record to the partial, and let it use whatever properties are needed. Otherwise the calling side needs to know which properties to pass. This breaks the principle of the separation of concerns.

By the way, in this case it's better to include the `audio` term in the partial's file name. Because we can have actions for other type of content.

Reply:
we can't pass whole record here as fileId specifier is different for series page (`fileId: audio.audios_id.id`) and audio page (`fileId: audio.fileId`)

Reply:
Can't we pass `audio` in one case and `audio.audio_id` in another?